Updating a pull request fails: Command failed with exit code 128

Hi!

Updating a remote branch seems to fail if it exists already - see [run1](https://github.com/Mudlet/Mudlet/actions/runs/55498515) where it was created successfully and [run2](https://github.com/Mudlet/Mudlet/runs/507423448?check_suite_focus=true) where updating it failed with:

```
Remote branch "update-texts-for-translators" checked out locally.
Branch "update-texts-for-translators" does not yet exist on remote.
##[error]Command failed with exit code 128 (Unknown system error -128): git checkout -b update-texts-for-translators
```

What could be wrong?

--- a/index.js
+++ b/index.js
@@ -248,9 +248,16 @@ async function checkOutRemoteBranch(branch) {
     await runShellCommand(`git checkout ${branch}`);
     core.info(`Remote branch "${branch}" checked out locally.`);
 
-    await runShellCommand(
-      `git cherry-pick --strategy recursive --strategy-option theirs ${TEMPORARY_BRANCH_NAME}`
-    );
+    try {
+      await runShellCommand(
+        `git cherry-pick --strategy recursive --strategy-option theirs ${TEMPORARY_BRANCH_NAME}`
+      );
+    } catch (error) {
+      // https://github.com/gr2m/create-or-update-pull-request-action/issues/245
+      if (/The previous cherry-pick is now empty/.test(error.stderr)) {
+        await runShellCommand(`git cherry-pick --skip`);
+      }
+    }
 
     return true;
   } catch (error) {

